packages/abi-gen-wrappers/package.json

```diff
@@ -18,7 +18,8 @@
         "prettier": "prettier --write src/**/* --config ../../.prettierrc",
         "prettier_contract_wrappers": "prettier --write src/generated-wrappers/* --config ../../.prettierrc",
         "clean": "shx rm -rf lib src/generated-wrappers",
-        "generate_contract_wrappers": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials/**/*.handlebars' --output src/generated-wrappers --backend ethers"
+        "generate_contract_wrappers": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials/**/*.handlebars' --output src/generated-wrappers --backend ethers",
+        "docs": "typedoc --excludePrivate --excludeExternals --target ES5 --tsconfig typedoc-tsconfig.json --out generated_docs ./src/generated-wrappers/*"
```

Comment:
I think we should somehow integrate this new `docs` script with both CI and the local development workflow.  Including it in `yarn build` seems reasonable enough for me, but there may be a better place. (Isn't there a `build-docs` step in CI?)

Reply:
There _is_ a `build-docs` step in CI, but @fabioberger is working on a totally new docs publishing flow which should be ready soon. We decided _not_ to add this one to the existing publishing flow, so it's in limbo right now until the new flow is ready. For now we only need to make sure that it's valid typedoc.

Reply:
Okay, that all makes sense, but I still think we should integrate this with CI somehow (`yarn build` is plenty sufficient), just so that this command doesn't go unexercised and break at some point without us knowing.